http/src/main/resources/reference.conf

```diff
@@ -85,8 +85,7 @@ dataproc {
 }
 
 gce {
-  customGceImage = "projects/broad-dsp-gcr-public/global/images/custom-leo-image-gce-debian9-5e2206c"
-  customGceSnapshot = "projects/broad-dsp-gcr-public/global/snapshots/custom-leo-gce-cos-snapshot-d1a330f"
+  customGceImage = "projects/broad-dsp-gcr-public/global/snapshots/custom-leo-gce-cos-image-704c032"
```

Comment:
Is updating this image to COS dependent on @Qi77Qi's PR?

(Should we stick with the `debian9` in this PR so we can merge it? 🤔 )

Reply:
I thought the part of Qi's work this image version depends on was merged but please let me know if that's not the case @Qi77Qi and I'll change it here.

Reply:
I think this won't work. Using COS based images require a bunch changes done in my PR

Reply:
Yeah I think this PR is needed in order for COS to work: https://github.com/DataBiosphere/leonardo/pull/2088

Reply:
Cool - the integration tests failed for that reason too. Will revert.